src/tests/test_transform.py

```diff
@@ -74,18 +74,17 @@ def test_isAddonNewer(self):
 		"""
 		Ensure only the newest addon is used for a version+channel.
 
-		getLatestAddons checks addons in order, and we want to ensure
-		the addon dictionary is updated to maintain the newest addon
-		for a version+channel.
+		getLatestAddons checks addons in order, and we want to ensure the addon dictionary is updated to
+		maintain the newest addon for a version+channel.
 
-		This tests _isAddonNewer by confirming that the order of addons being added
-		does not effect the results.
+		This tests that the newest addons is always stored by confirming that the order of addons supplied
+		does not effect the output.
```

Comment:
Should there be a similar test to verify the order of the NVDA API Version data?

Reply:
I don't really see what that test would involve or why the ordering of the NVDA API Version matters, as it's not used in any comparisons.

Reply:
Just thinking about the function stand-alone. The function takes two lists, and outputs a dictionary. [Dictionaries in python 3.7 are ordered](https://mail.python.org/pipermail/python-dev/2017-December/151283.html).

The things the caller can control are the contents of the two lists, and the order of the contents of the two lists. If for some reason the order of the API versions changes, does the output change?

Reply:
Does the order of the output dictionary matter? I would be surprised if `assertDictEqual` checks the ordering.

Reply:
The following test passes:

```py
	def test_assertDictEq(self):
		d1 = {}
		d1["first"] = 1
		d1["second"] = 2
		d2 = {}
		d2["second"] = 2
		d2["first"] = 1
		self.assertDictEqual(d1, d2)
```

Reply:
Interesting, it probably should fail, given the order of the dict may now be something that is relied upon. I guess that's the danger with changing dicts to be ordered.

Reply:
Is this something we need to rely upon?